modules/conversion-center/assets/js/frontend/handlers/contact-buttons.js

```diff
@@ -23,6 +28,7 @@ export default class ContactButtonsHandler extends Base {
 				hasAnimations: 'has-animations',
 				hasEntranceAnimation: 'has-entrance-animation',
 				none: 'none',
+				isEditor: 'elementor-editor-active',
```

Comment:
You can use `elementorFrontend.isEditMode()` throughout.

Reply:
@louiswol94 done and pushed